client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -196,18 +226,51 @@ class CampaignAdminList extends SilverStripeComponent {
           <Toolbar showBackButton handleBackButtonClick={this.props.handleBackButtonClick}>
             <Breadcrumb multiline />
           </Toolbar>
+          {newItemInfo}
           <div className={bodyClass.join(' ')}>
             {body}
           </div>
           <div className="toolbar toolbar--south">
             {this.renderButtonToolbar()}
           </div>
         </div>
-        <Preview itemLinks={itemLinks} itemId={itemId} onBack={this.handleCloseItem} />
+        {this.renderPreview(itemLinks, itemId)}
       </div>
     );
   }
 
+  renderPreview(itemLinks, itemId) {
+    let preview = null;
+    let previewClasses = classnames([
+      'flexbox-area-grow',
+      'fill-height',
+      'preview',
+      'campaign-admin__campaign-preview',
+      'campaign-admin__campaign-preview--empty',
+    ]);
+
+    if (this.state.loading) {
+      preview = (
+        <div className={previewClasses}>
+          <p>Loading...</p>
+        </div>
+      );
+    } else if (!this.getItems() || this.getItems().length === 0) {
+      preview = (
+        <div className={previewClasses}>
+          <h2 className="campaign-admin__empty-heading">Getting started</h2>
+          <p className="campaign-admin__empty-info">
+            Select <strong>Add to Campaign</strong> from pages, files, and other content types
```

Comment:
be careful with adding html to translation strings, it's heavily discouraged because you need to use `dangerouslySetInnerHTML` in react... (or `castStringToElement()`)
Can we use quotes instead?
I've raised this about the "Nice one!" being bold above as well, and it was agreed that we could remove the bold for that.